rc/base/lint.kak

```diff
@@ -8,6 +8,8 @@ decl -hidden range-specs lint_errors
 
 def lint -docstring 'Parse the current buffer with a linter' %{
     %sh{
+        export PATH="${PATH}:${kak_runtime}/sh"
```

Comment:
I think a '# mktemp fallback' comment should follow, to make it explicit which fallback we try to import